src/caja-query-editor.c

```diff
@@ -999,6 +1025,231 @@ type_add_rows_from_query (CajaQueryEditor    *editor,
 
 /* End of row types */
 
+
+static GtkWidget *modtime_row_create_widgets(CajaQueryEditorRow *row)
+{
+    GtkWidget *hbox = NULL;
+    GtkWidget *combo = NULL;
+    GtkWidget *duration_combo = NULL;
+    GtkCellRenderer *cell = NULL;
+    GtkListStore *store = NULL;
+    GtkListStore *duration_store = NULL;
+    GtkTreeIter iter;
+
+    hbox = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, 7);
+
+    store = gtk_list_store_new(2, G_TYPE_BOOLEAN, G_TYPE_STRING);
+    combo = gtk_combo_box_new_with_model(GTK_TREE_MODEL(store));
+    g_object_unref(store);
+
+    cell = gtk_cell_renderer_text_new();
+    gtk_cell_layout_pack_start(GTK_CELL_LAYOUT(combo), cell, TRUE);
+    gtk_cell_layout_set_attributes(GTK_CELL_LAYOUT(combo), cell, "text", 1,
+                                   NULL);
+
+    gtk_list_store_append(store, &iter);
+    gtk_list_store_set(store, &iter, 0, FALSE, 1, _("Less than or equal to"), -1);
+    gtk_list_store_append(store, &iter);
+    gtk_list_store_set(store, &iter, 0, TRUE, 1, _("Greater or equal to"), -1);
+
+    gtk_combo_box_set_active(GTK_COMBO_BOX(combo), 0);
+
+    duration_store = gtk_list_store_new(2, G_TYPE_LONG, G_TYPE_STRING);
+    duration_combo = gtk_combo_box_new_with_model(GTK_TREE_MODEL(duration_store));
+    g_object_unref(duration_store);
+
+    gtk_cell_layout_pack_start(GTK_CELL_LAYOUT(duration_combo), cell, TRUE);
+    gtk_cell_layout_set_attributes(GTK_CELL_LAYOUT(duration_combo), cell,
+                                   "text", 1, NULL);
+
+    gtk_list_store_append(duration_store, &iter);
+    gtk_list_store_set(duration_store, &iter, 0, 3600, 1, _("1 Hour"), -1);
+    gtk_list_store_append(duration_store, &iter);
+    gtk_list_store_set(duration_store, &iter, 0, 86400, 1, _("1 Day"), -1);
+    gtk_list_store_append(duration_store, &iter);
+    gtk_list_store_set(duration_store, &iter, 0, 604800, 1, _("1 Week"), -1);
+    gtk_list_store_append(duration_store, &iter);
+    gtk_list_store_set(duration_store, &iter, 0, 2419200, 1, _("1 Month"), -1);
+    gtk_list_store_append(duration_store, &iter);
+    gtk_list_store_set(duration_store, &iter, 0, 14515200, 1, _("6 Months"), -1);
+    gtk_list_store_append(duration_store, &iter);
+    gtk_list_store_set(duration_store, &iter, 0, 29030400, 1, _("1 Year"), -1);
```

Comment:
These values seem wrong. Here's what I noticed:
🗸 3600: 1 hour
🗸 86400: 1 day, 24 hours
🗸 604800: 7 days, 168 hours
🗴 2419200: 28 days, arguably correct?
🗴 14515200: 168 days, definitely wrong (should be around 15778800, 31557600 / 2, or about 183 days)
🗴 29030400: 336 days, definitely wrong (should be around 31557600, 365.25 days)